src/Network.h

```diff
@@ -73,6 +74,7 @@ class Network {
 
     static std::vector<net_t> gather_features(const GameState* const state,
                                               const int symmetry);
+    static std::pair<int, int> get_symmetry(const std::pair<int, int>& vertex, const int symmetry, const int board_size = BOARD_SIZE);
```

Comment:
I think line length has been kept to under 80 so far. There's a few other new places this has been exceeded.